tests/unit/test_serialization.py

```diff
@@ -349,7 +353,7 @@ def split_deserialized_dataset_into_individual_scenario(scenario_idx, deserializ
 def assert_not_a_value(value: np.ndarray):
     """Assert value contains only NaN-like values.
 
-    Depending on type, maps to np.nan, -127, na_IntID, ...
+    Depending on type, maps to np.nan, -127, na_IntID, ComponentAttributeFilterOptions
```

Comment:
gotta love global search/replace

```suggestion
    Depending on type, maps to np.nan, -127, na_IntID, ...
```
or
```suggestion
    Depending on type, maps to np.nan, -127, na_IntID, etc.
```

Reply:
Addressed.